lightstep/resource_metric_condition.go

```diff
@@ -561,6 +562,7 @@ func buildQueries(queriesIn []interface{}) ([]client.MetricQueryWithAttributes,
 
 		spansQuery := query["spans"]
 		if spansQuery != nil && len(spansQuery.([]interface{})) > 0 {
+			hasSpanSingle = true
```

Comment:
not new code, but on 564 is there any concern of the type assertion panicking?

Reply:
I think this is fine because of the provider type that we defined ensures this will be an array. Honestly though, I'm guessing here. Seeing as it hasn't bitten us yet, I'm not too worried about this.